fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -469,6 +470,7 @@ public class FunctionSet {
                     .add(FunctionSet.NOW)
                     .add(FunctionSet.UTC_TIMESTAMP)
                     .add(FunctionSet.MD5_SUM)
+                    .add(FunctionSet.MD5_SUM_NUMERIC)
```

Comment:
should be nullable

Reply:
> should be nullable

This is to be the same as the implementation of md5sum, the result of `select md5sum(null)` is not `NULL`, I think the behavior of md5sum_numeric should be consistent with it